scripts/cli/updates/1.4.0.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env
+
+source $(pwd)/keg
+
+source $(pwd)/scripts/cli/updates/update_helpers.sh
+
+keg_reset_default_env(){
+  local KEG_DEFS=~/.kegConfig/defaults.env
+  if [[ -f "$KEG_DEFS" ]]; then
+    rm -rf ~/.kegConfig/defaults.env
+  fi
+
+  cp $(pwd)/scripts/setup/defaults.env ~/.kegConfig/defaults.env
+}
+
+keg_cli_1_4_0_update(){
+
+  keg_message "Running update for version 1.4.0..."
+
+  keg_reset_default_env
+
+  local RUNNING_CONT="$(command docker ps -aq)"
+  if [[ "$RUNNING_CONT" ]]; then
+    keg_error "All docker containers must be stopped before updating!"
+    return
+  fi
+  
+  keg_docker_clean "$@"
+
+  # Resync the global config
+  keg config sync --confirm false
+  keg config set --key cli.settings.checkUpdates --value true --confirm false
```

Comment:
* This line is not needed. Nothing wrong with having it, but is someone turns updates off, this will just turn it back on again